### ISSUE TITLE
Uncased versions for multi-lingual models

### ISSUE THREAD

**Issue by thejaswiraya** [UKPLab/sentence-transformers#1837]
I noticed that all the multi-lingual pre-trained models made available via sentence-transformers are cased versions. 

1. Why is that? 
2. Are there uncased versions of these models? For example: uncased version of `sentence-transformers/paraphrase-multilingual-minilm-L12-v2`
3. Based on the documentation and my observation, `model.encode` does not have a way to lowercase sentences. I understand that the `AutoTokenizer` has `do_lower_case` parameter to lowercase sentences, but is there a way to do it using sentence-transformers? I have a hack (see below) to do that, but I am trying to see if there's a better way. 
```
model_name = 'sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2' 
model = SentenceTransformer(model_name) 
model[0].do_lower_case = True`
```
4. I am fine-tuning `sentence-transformers/paraphrase-multilingual-minilm-L12-v2` embeddings model on my custom dataset and I see that there are cases where the casing of a string makes a big difference to the embeddings. For my dataset, this should not be the case. I am currently resorting to lowercasing the sentences in my custom dataset and fine-tuning `paraphrase-multilingual-minilm-L12-v2`. However, the accuracy of the model is worse than just using the case as-is which in hindsight is understandable. How do I go about getting a better model performance with the uncased version because for my dataset case does not matter? 

Much appreciated.
